doc/03-Configuration.md

```diff
@@ -0,0 +1,28 @@
+# Configuration
+
+This is how you configure your Icinga DB so it works right off the bat.
+
+## Database
+
+Select the database resource for your setup.
+
+## Redis
+
+Configure the Redis connections details of your Icinga host.  
```

Comment:
- s/connections/connection/
- Explain or link where the Redis component is described in detail. Which service or package is responsible for setting this up?